.github/workflows/ipc_fuzzer.yml

```diff
@@ -66,4 +66,11 @@ jobs:
             IPC3) cmake_arg='-DCONFIG_IPC_MAJOR_3=y' ;;
             IPC4) cmake_arg='-DCONFIG_IPC_MAJOR_4=y' ;;
           esac
-          sof/scripts/fuzz.sh -o _.log -t 300 -- "$cmake_arg"
+          sof/scripts/fuzz.sh -o fuzz-stdout.txt -t 300 -- "$cmake_arg"
+
+      - name: Upload stdout
+        uses: actions/upload-artifact@v3
+        with:
+          name: ${{ matrix.IPC }} logs
```

Comment:
do we maybe want to compress these? from this run it was ~70MB also what is the lifespan of the files before autodelete?